index.html

```diff
@@ -3559,6 +3559,34 @@ <h3>
           Changes since 14 July 2016
```

Comment:
Changes since [01 June 2017](https://www.w3.org/TR/2017/CR-presentation-api-20170601/)

And move ”Changes since [14 July 2016](https://www.w3.org/TR/2016/CR-presentation-api-20160714/)” with a link added to L3590.

Reply:
I split the list into two as suggested, and added PRs #486 and #487 which I just merged.  I think this should be up-to-date now.